plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/intermediate/greedyswitch/InLayerEdgeTwoNodeCrossingCounter.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014, 2015 Kiel University and others.
+ * Copyright (c) 2016 Kiel University and others.
```

Comment:
I think 2014, 2016 would be more appropriate, even when only about 10 lines of code remain from the previous implementation.